src/Navigation/Header/styled.js

```diff
@@ -0,0 +1,92 @@
+import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+import { ReactComponent as LogoWebsite } from "./logo.svg"
+
+export const StyledHeader = styled.header`
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
It seems we have many properties that don't do anything? Like `justify-content: center` in this case?
At the end please remember to remove them 😃